omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1616,7 +1697,8 @@ def run_script():
     """
 
     export_options = [rstring('PDF'), rstring('PDF_IMAGES'),
-                      rstring('TIFF'), rstring('TIFF_IMAGES')]
+                      rstring('TIFF'), rstring('TIFF_IMAGES'),
+                      rstring('OMERO')]
```

Comment:
For the others ```PDF_IMAGES/ TIFF_IMAGES``` means create a zip file with the Figure as a PDF/TIFF and include all the rendered images in the figure as TIFFs (so you have a record of how the figure was created on export and you can use them to tweak the figure later). We don't need to support the equivalent option for export to OMERO.